public/js/graph/react_graph.js

```diff
@@ -612,6 +615,7 @@ var Node = React.createClass({
                     var sourceNode = svg.refs['nodes'].refs[currentEdge.props.source] ||
                                      svg.refs['bools'].refs[currentEdge.props.source];
                     if (!sourceNode.isSelected()) {
+                        // currentEdge.setState({status: 'missing'}, () => svg.refs['edges'].render());
```

Comment:
Please delete this (and remove other commented-out blocks that are no longer necessary)